### ISSUE TITLE
[BUG] View Image button not working. Can only right-click, not left-click?

### ISSUE THREAD

**Issue by gaveitatry2** [fanfare/googleimagesrestored#96]
Hello, I have some questions about the Google Images Restored.

For a long time now, not sure, but maybe at least six months... when I click on an image in Google Image Search Results, it shows two buttons. One is the Visit button and one is the View Image button. If I left-click the View Image button nothing happens. If I hold down CTRL and left-click the View Image button nothing happens. The only way to open up the image in a new tab is to right-click the View Image button and select 'Open Link in New Tab'. Ideally, I would like to be able to just left-click or CTRL + left-click the button to open up the image in a new tab.

I am using version 0.5.5 on Mozilla Firefox.

**Comment by fanfare** [comment]
im unfortunately unable to reproduce this on my computer using firefox and 0.5.5.  all i can say is it is either some sort of a/b testing situation or perhaps maybe there is another extension that is conflicting with this googleimagesrestored extension.

**Comment by gaveitatry2** [comment]
@fanfare 

Here is a video demonstrating that it is not another extension conflicting with Google Images Restored.

https://github.com/fanfare/googleimagesrestored/assets/75463665/3290ad00-4a64-4111-a512-701b52c0047e

**Comment by fanfare** [comment]
does signing out of google change/fix anything? or, if the extension can be allowed to open in private mode, does the same thing happen in private mode as well? what version of firefox are you using also

**Comment by gaveitatry2** [comment]
> does signing out of google change/fix anything? or, if the extension can be allowed to open in private mode, does the same thing happen in private mode as well? what version of firefox are you using also

I don't know what version of Firefox I was using yesterday, but today I updated to version 116.0 and the same problem exists. I've also had this problem for many, many months. Even after a clean install of Firefox. I can't remember how long exactly. Could be six months. Could be even longer. I did try signing out of Google just now. That did not affect anything. I also tried opening a Private Window (after changing permissions to be allowed to open in private mode), and that also did not change anything.
